api/api.go

```diff
@@ -101,7 +107,69 @@ func (api *BlockChainAPI) SendRawTransaction(
 	ctx context.Context,
 	input hexutil.Bytes,
 ) (common.Hash, error) {
-	return crypto.Keccak256Hash([]byte("hello world")), nil
+	gethTx := &types.Transaction{}
+	encodedLen := uint(len(input))
+	err := gethTx.DecodeRLP(
+		rlp.NewStream(
+			bytes.NewReader(input),
+			uint64(encodedLen),
+		),
+	)
+	if err != nil {
+		return common.Hash{}, err
+	}
+
+	block, err := api.FlowClient.GetLatestBlock(context.Background(), true)
+	if err != nil {
+		return common.Hash{}, err
+	}
+
+	privateKey, err := sdkCrypto.DecodePrivateKeyHex(sdkCrypto.ECDSA_P256, strings.Replace("2619878f0e2ff438d17835c2a4561cb87b4d24d72d12ec34569acd0dd4af7c21", "0x", "", 1))
```

Comment:
can you get this from ENV configuration

Reply:
I am planning to add a `flow.json` config file, to hold the account and private key that will be used from the gateway. Though I didn't get to it in this PR, to avoid cluttering things. Should we add it here?

Reply:
I see. Just add a todo then maybe.

Reply:
Sure thing :+1: I also want to add the networks in there, where the indexer will connect to. And we'll see what other config makes sense to have in there.

Reply:
Added comments in https://github.com/onflow/flow-evm-gateway/pull/26/commits/4ad8061a27715e5e8dd8a8bde399efa177b3f509